common/lib/util/eventemitter.js

```diff
@@ -41,7 +41,7 @@ var EventEmitter = (function() {
 	 *        supplied, all listeners are removed.
 	 */
 	EventEmitter.prototype.off = function(event, listener) {
-		if(arguments.length == 0) {
+		if(arguments.length == 0 || (!event && !listener)) {
```

Comment:
Yes, true.
I am a little uncomfortable with the explicit checks for `null` in some cases, and `undefined` in others.  @paddybyers do you have any reason to not simply evaluate the arguments as truthy like I have done in this line throughout?